Tests/EventDispatcherTests.swift

```diff
@@ -0,0 +1,198 @@
+import Quick
+import Nimble
+import Clappr
+
+class EventDispatcherTests: QuickSpec {
+
+    override func spec() {
+        describe("EventDispatcher") {
+
+            var eventDispatcher: EventDispatcher!
+            var callbackWasCalled: Bool!
+
+            let eventName = "some-event"
+            let callback: EventCallback = { _ in
+                callbackWasCalled = true
+            }
+
+            beforeEach {
+                eventDispatcher = EventDispatcher()
+                callbackWasCalled = false
+            }
+
+            describe("on") {
+                it("executes callback function") {
+                    eventDispatcher.on(eventName, callback: callback)
+                    eventDispatcher.trigger(eventName)
+
+                    expect(callbackWasCalled) == true
+                }
+
+                it("receives userInfo when triggered with params") {
+                    var value = "Not Expected"
+                    eventDispatcher.on(eventName) { userInfo in
+                        value = userInfo?["new_value"] as! String
+                    }
+
+                    eventDispatcher.trigger(eventName, userInfo: ["new_value": "Expected"])
+
+                    expect(value) == "Expected"
+                }
+
+                it("executes multiple callback functions") {
+                    eventDispatcher.on(eventName, callback: callback)
+
+                    var secondCallbackWasCalled = false
+                    eventDispatcher.on(eventName) { _ in
+                        secondCallbackWasCalled = true
+                    }
+
+                    eventDispatcher.trigger(eventName)
+
+                    expect(callbackWasCalled) == true
+                    expect(secondCallbackWasCalled) == true
+                }
+
+                it("doesn't execute callback for another event") {
+                    eventDispatcher.on(eventName, callback: callback)
+
+                    eventDispatcher.trigger("another-event")
+
+                    expect(callbackWasCalled) == false
+                }
+
+                it("doesn't executes callback for another context object") {
+                    let anotherDispatcher = EventDispatcher()
+
+                    eventDispatcher.on(eventName, callback: callback)
+
+                    anotherDispatcher.trigger(eventName)
+
+                    expect(callbackWasCalled) == false
+                }
+            }
+
+            describe("once") {
```

Comment:
According to the Project conventions, we should add the # prefix on the Method Name.

``context("#once")``